lib/plugins/aws/package/compile/events/apiGateway/lib/method/integration.js

```diff
@@ -88,6 +88,12 @@ module.exports = {
       });
     }
 
+    if (type === 'AWS' || type === 'HTTP') {
+      _.assign(integration, {
+        RequestParameters: this.getIntegrationRequestParameters(http),
```

Comment:
You set the RequestParameters property unconditionally here - so, even if I do not specify one (intentionally) it gets set. According to the AWS REST API documentation, the property should be omitted if it is not used.

Imo the property should only be set in `integration` if the user specified it.